tools/lisa-build-asset

```diff
@@ -8,8 +8,45 @@ import shutil
 import sys
 
 from lisa.utils import nullcontext
+from contextlib import contextmanager
+from shlex import quote
 
 LISA_HOME = os.environ['LISA_HOME']
+ARCHITECTURES = ['arm64', 'armeabi', 'x86_64']
+
+# Arch names usued by qemu.
+QEMU_ARCH_NAMES = {
+    'arm64': 'aarch64',
+    'x86_64': 'x86_64'
+}
+
+def get_alpine_dir():
+    # Locate where 'alpine-chroot-install' was installed.
+    return os.path.join(os.environ['LISA_HOME'], 'tools', 'alpine-chroot-install')
+
+def support_native_build(asset, recipe_dir, arch):
+    # Not all assets/arch couple support native building.
+    # Check that the arch is suported
+    if not QEMU_ARCH_NAMES.get(arch, None):
+        return False
+
+    # Read the recipe, searching for the 'ALPINE_BUILD_DEPENDENCIES' definition.
```

Comment:
That should be split separately, as building under QEMU is orthogonal to building on Alpine.